poker-pals/src/components/NavBar.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <div class="navigationTab">
```

Comment:
I'm currently working on the nav bar. Not sure if we want to remove or merge these changes

Reply:
my version is just space holder for formatting the web page its not to be used in the final design